src/main/scala/format/pgn/Parser.scala

```diff
@@ -20,7 +20,7 @@ object Parser extends scalaz.syntax.ToTraverseOps {
   )
 
   def full(pgn: String): Valid[ParsedPgn] = try {
-    val preprocessed = pgn.lines.map(_.trim).filter {
+    val preprocessed = augmentString(pgn).lines.map(_.trim).filter {
```

Comment:
what is this about?

Reply:
https://github.com/scala/bug/issues/11125#issuecomment-423375868